### DISCUSSION TITLE
Recommended extensions?

### DISCUSSION THREAD

**Discussion by iainmwallace** [posit-dev/positron#3754] [Q&A]
I came across this very helpful blog post and was wondering what other extensions people might recommend 

https://christophertkenny.com/posts/2024-06-29-positron-settings/

Personally, I find the continue extension for AI assisted coding works really well with positron
Thanks

Iain

**Comment by juliasilge** [comment]
One that might be helpful for you (and isn't in that blog post) is the [Shiny extension](https://open-vsx.org/extension/posit/shiny), which has support for Shiny in both R and Python.

**Comment by chuxinyuan** [comment]
[vscode-pdf](https://github.com/tomoki1207/vscode-pdfviewer), allows you to immerse yourself in reading literature and writing papers.
